gm4_better_armor_stands/data/better_armor_stands/functions/main.mcfunction

```diff
@@ -0,0 +1 @@
+execute as @e[type=item,nbt={Item:{id:"minecraft:writable_book"}}] at @s if entity @e[type=armor_stand,distance=..1,limit=1,sort=nearest] run function #better_armor_stands:update
```

Comment:
Adding an `OnGround:1b` tag to the nbt might be smart, it'll ensure to only edit the stand once the item isn't falling anymore. E.g. `nbt={Item:{id:"minecraft:writable_book"},OnGround:1b}`